anchore_engine/configuration/localconfig.py

```diff
@@ -101,6 +101,17 @@
 CRED_CACHE_TTL = int(os.getenv("ANCHORE_INTERNAL_CRED_CACHE_TTL", 600))
 CRED_CACHE_LOCK_WAIT_SEC = int(os.getenv("ANCHORE_INTERNAL_CRED_CACHE_WAIT_SEC", 3))
 
+ANALYZER_SEARCH_PATHS = ["anchore_engine.analyzers"]
+
+
+def register_analyzers(module_path):
+    global ANALYZER_SEARCH_PATHS
+    ANALYZER_SEARCH_PATHS.append(module_path)
+
+
+def analyzer_paths():
+    return ANALYZER_SEARCH_PATHS
```

Comment:
I'm not following why this needs to be dynamic via functions that need to use `global`.

Reply:
so that it can be modified if needed during bootstrap of the service based on external config. the analyzer service uses it but so does the `anchore-manager analyzers list` command that doesn't have a service bootstrap